### ISSUE TITLE
Indexed x axis doesn't scale as expected

### ISSUE THREAD

**Issue by netil** [naver/billboard.js#877]
## Description

Isn't correctly scaling well when indexed `x` axis values are given.
Works well in previous version, possibly caused by changes from 1.8.0

## Steps to check or reproduce
```js
var chart = bb.generate({
  data: {
    x: "x",
    columns: [
        ["x", 495, 740, 1500, 3000, 4500, 6000, 7500, 9000, 10500, 12000, 13500, 15000],
        ["data", 47.911, 47.915, 48.437, 49.117, 49.583, 50.28, 51.712, 53.103, 54.456, 55.955, 56.752, 56.851]
    ]
  }
});
```

**Comment by netil** [comment]
Is a side effect caused by #714.
`sortValue()` added to util.js is wrongly sorting when x Axis is indexed in reverse way.

**Comment by netil** [comment]
released on 1.8.1
